docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -546,7 +546,7 @@ You can also set `exec_timeout_secs` using [timeout.update](Project-Commands#tim
 **Idle timeout: timeout_secs**
 You may also force a specific command to trigger a failure if it does not appear
 to generate any output on `stdout`/`stderr` for more than a certain threshold,
-using the `timeout_secs` setting on the command. As long as the command produces
+using the `timeout_secs` setting on the command, or the overall project. As long as the command produces
```

Comment:
These changes look good. However, thinking about it more, there is nothing preventing someone from setting it to 30 days and keeping idle tasks just sitting there for an amount of time that ends up costing a lot of money for very little gain. This can happen by mistake too, (at a certain point, one extra zero goes from .8 weeks to 8 weeks). 

Allowing setting it on the project level increases how bad the potential effect is if it does happen. It might be a good idea to use this opportunity to introduce a maximum.

Reply:
That sounds reasonable, although I think it'd be best to spend dedicated time in a separate ticket querying through all projects and determining what the max timeout_secs value is so that we don't break any builds by enforcing a value below what's already set in some projects. For example, just looking at server's project there's an instance of them setting it to [7 days](https://gist.github.com/hadjri/3d7741249ecd20c16f9851c9a626932e#file-misc_tasks-yml-L924).

And we'd want to make sure that we do necessary comms and warnings if we decide it's beneficial to us to rollout a change that requires users to restrict some of their `timeout_secs` values.

Reply:
that sounds good, thank you!